streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -489,7 +490,11 @@ private void commitAll() {
     private void commitOne(AbstractTask task, long now) {
         try {
             task.commit();
+        } catch (CommitFailedException e) {
```

Comment:
```CommitFailedException``` can be thrown not only for rebalance-in-progress, but for any non-retriable commit failure due to incorrect membership. Could we change the comment to ".. due to incorrect membership. Just log it since the next commit will likely to succeed after re-joining the group".